internal/support/scheduler/application/scheduler/manager.go

```diff
@@ -134,7 +141,16 @@ func (m *manager) executeAction(action models.IntervalAction) errors.EdgeX {
 		if !ok {
 			return errors.NewCommonEdgeX(errors.KindContractInvalid, "fail to cast Address to RESTAddress", nil)
 		}
-		_, err := utils.SendRequestWithRESTAddress(m.lc, action.Content, action.ContentType, restAddress)
+
+		var jwtSecretProvider clientInterfaces.AuthenticationInjector
+		// Awaiting change to go-mod-core-contracts
```

Comment:
Is this still valid? Waiting foe what? Should start with a `TODO:` so readers know it needs to be addressed.

Reply:
Changed the code without removing the comment.  Yes, this is done.

Reply:
Misleading comment removed in https://github.com/edgexfoundry/edgex-go/pull/4419/commits/fac7cbe8b63e9dd8d2681f2850907dd0c628a974